src/tbp/monty/frameworks/utils/spatial_arithmetics.py

```diff
@@ -135,8 +135,8 @@ def get_angles_for_all_hypotheses(hyp_f, query_f):
     return shape = (num_hyp, num_nn)
 
     Args:
-        hyp_f (num_hyp, num_nn, 3): ?
-        query_f (num_hyp, 3): ?
+        hyp_f: ?
+        query_f: ?
```

Comment:
This entire docstring is _almost_ useless. Not sure who wrote it, but I think we should try to fill in the blanks.